flowtools/peacoqc/PeacoQC_config.xml

```diff
@@ -0,0 +1,27 @@
+<tool id="peacoqc_tool" name="PeacoQC Analysis" version="0.1.0">
+    <description>Flow data QC using PeacoQC</description>
+    <requirements>
+        <requirement type="package" version="1.0">r-base</requirement>
+        <requirement type="package" version="1.8.0">bioconductor-peacoqc</requirement>
+        <!-- Include other dependencies such as flowCore here -->
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript --vanilla $__tool_directory__/peacoqc_script.R '$input' '$output'
+    ]]></command>
+    <inputs>
+        <param name="input" type="data" format="fcs" label="Input FCS file" help="FCS file containing flow cytometry data for quality control."/>
+    </inputs>
+    <outputs>
+        <data name="output" format="rdata" label="${tool.name} on ${on_string}: QC Results"/>
```

Comment:
Here you can add more data outputs, supposing for instance that we have a file created in the working directory named "heatmap.pdf", we would add something like:

```
<data name="heatmap_plot" format="pdf" label="${tool.name} on ${on_string}: Heatmap" from_work_dir="heatmap.pdf"/>
```

Further documentation at https://docs.galaxyproject.org/en/latest/dev/schema.html